packages/react-components/src/templates/__tests__/GroupProfileAbout.test.tsx

```diff
@@ -58,3 +58,28 @@ it('assigns given id to the members section for deep linking', () => {
     /members/i,
   );
 });
+
+it('renders a call to action button, when a PM is defined on the group', () => {
```

Comment:
We should also test that the CTA card is not shown when there are no project managers

Reply:
I actually wrote the test, and deleted it... I saw the case not being tested on the Team Page...